nav-bar.css

```diff
@@ -0,0 +1,54 @@
+.nav-bar {
+  display: flex;
+  background-color: var(--white);
+  border: 3px solid var(--lightgray);
+  gap: 0.9rem;
+  padding: 0.4rem;
+  align-items: center;
+  flex-wrap: wrap;
+  margin: 0.6rem;
+  justify-content: space-around;
+}
+.navbar-icons {
+  display: flex;
+  gap: 2.2rem;
+}
+.nav-inp {
+  padding: 0.6rem;
+  width: 20%;
+  outline: 2px solid transparent;
+  outline-offset: 2px;
```

Comment:
you can extract this in a common class since it will be used for almost every button on your webpage

Reply:
Okay